build.sh

```
@@ -0,0 +1,59 @@
+
+dotnet restore "$solutionPath"
+
+function sync_mod() {
+    # Copy over the mod directory.
+    rsync -a ${MOD} /rimworld/1.2/Mods/
+
+    # Copy over and reformat the README.
+    cp README.md /rimworld/1.2/Mods/${MOD}
+    unix2dos /rimworld/1.2/Mods/${MOD}/README.md
+
+    rm -rf /rimworld/1.3/Mods/${MOD}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)